doc/dev_guide/internals/iproto/requests.rst

```diff
@@ -126,6 +130,18 @@ constants for return codes.
 To learn more about error responses,
 check the section :ref:`Request and response format <box_protocol-responses_error>`.
 
+..  _internals-iproto-unknown:
+
+IPROTO_UNKNOWN
+~~~~~~~~~~~~~~
+
+Since :doc:`2.11.0 </release/2.11.0>`.
+
+Code: -1.
+
+Unknown request type. The constant is used for overriding an unknown IPROTO request handler.
```

Comment:
`Unknown request handler` sounds weird. This is a constant for overriding the handler for unknown IPROTO request types.